app/assets/stylesheets/application.css

```diff
@@ -121,3 +121,25 @@
   margin: 10px;
   margin-bottom: 20px;
 }
+
+.login .loginn {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.signup {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+.login {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
```

Comment:
| @Ouail-01 Design | Microverse Design |
|-|-|
| ![image](https://github.com/Ouail-01/Blog-App/assets/93039359/e3ade4e0-bf3c-423a-8a9c-e8a9ab3173ba) | ![image](https://github.com/Ouail-01/Blog-App/assets/93039359/b51c143b-25a1-47a8-a5dd-b2daafa2826d) |

- [ ] Great work so far on making the changes but it would be great if you follow the given design by microverse as suggested by the previous reviewer 👍 Kindly make your input fields and button expand on the whole screen of both `Login` and `Sign up` page to follow the given design 😊